packages/pluggableWidgets/bubble-chart-web/src/utils/index.ts

```diff
@@ -0,0 +1,34 @@
+import { LinesType } from "../../typings/BubbleChartProps";
+import { Dimensions } from "@mendix/piw-utils-internal";
+
+export const getMarkerSizeReference = (props: LinesType, markerSize: number[], dimensions?: Dimensions): number => {
+    if (props.autosize) {
+        const width = dimensions ? dimensions.width : 0;
+        const height = dimensions ? dimensions.height : 0;
+        let sizeRef = 1;
+        const averageSize = (width + height) / 2;
+        const percentageSize = averageSize / (1 / (props.sizeref / 100));
+
+        if (markerSize.length > 0) {
+            sizeRef = Math.max(...markerSize) / percentageSize;
+        }
+
+        return Math.round(sizeRef * 1000) / 1000;
```

Comment:
Shouldnt this be times 100 and divided by 100?

Reply:
To be honest, I don't know. I found this functionality on the previous charts.
[LineChart](https://github.com/mendix/charts/blob/5f14b86facbb05ff086546faeb4347bca0912523/src/LineChart/utils/data.ts#L70)

Reply:
Seems weird to calculate percentage with 1000..

Reply:
let me check diff in sizes, i'll put screenshots here

Reply:
sizeref before `return` looks like` xx.xxxxxxxxxxxxxx` and dividing it by 1000 just changes how long value will be after dot. if it's 1000 then the result will be `xx.xxx` if 100 then `xx.xx`

Reply:
No need for screenshots because it doesn't change a visual

Reply:
Ah ok, so about precision